server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const app = express();
+
+app.use(express.static(__dirname + '/dist/'));
+
+app.get('*', (req, res) => {
+  res.sendFile(__dirname + '/dist/');
+});
+
+app.listen(3000, () => console.log('We are on port 3000'));
```

Comment:
```suggestion
const PORT = process.env.PORT || 3000;
app.listen(PORT, () => console.log('We are on port 3000'));
```

Часто нужно указать порт из переменной окружения